app/code/Magento/MediaGallery/Plugin/Wysiwyg/Images/Storage.php

```diff
@@ -92,4 +99,36 @@ public function afterDeleteFile(StorageSubject $subject, StorageSubject $result,
 
         return $result;
     }
+
+    /**
+     * Delete media data after the folder delete action from Wysiwyg
+     *
+     * @param StorageSubject $subject
+     * @param null $result
+     * @param string $path
+     *
+     * @return null
+     *
+     * @SuppressWarnings(PHPMD.UnusedFormalParameter)
+     */
+    public function afterDeleteDirectory(StorageSubject $subject, $result, $path)
+    {
+        if (!is_string($path)) {
+            return $result;
+        }
+
+        $relativePath = $this->filesystem->getDirectoryRead(DirectoryList::MEDIA)->getRelativePath($path);
+
+        if (!$relativePath) {
+            return $result;
+        }
+
+        try {
+            $this->deleteMediAssetByDirectoryPath->execute($relativePath);
+        } catch (\Exception $exception) {
+            $this->logger->critical($exception);
```

Comment:
You are logging the same exception twice?
https://github.com/magento/magento2/pull/26015/files#diff-01fc33f90d1b68e3f97bbed6e144e565R72